features/support/files_helpers.rb

```diff
@@ -23,6 +23,32 @@ def files_in_branches
 end
 
 
+def create_uncommitted_file options = {}
+  options.reverse_merge!(default_uncommitted_file_attributes)
+  IO.write options[:file_name], options[:file_content]
+  options[:file_name]
+end
+
+
+def default_uncommitted_file_attributes
+  {
+    file_name: "default_uncommitted_file_name_#{SecureRandom.urlsafe_base64}",
```

Comment:
What do you think about naming the uncommitted file "uncommitted_file" by default? I don't see a need for the random hash, since we won't create two uncommitted files at the same time. And if the file name ever matters in a test (because we want to test for collisions), the test must specify it explicitly anyways.